e2e/helpers.py

```diff
@@ -121,11 +121,7 @@ def ensure_appropriate_publishers_and_sign(self, fake_profile, condition):
             else:
                 if attr != "schema" and attr in complex_structures:
                     for k in temp_profile[attr]:
-                        if attr == "access_information":
-                            successful_random_publisher = random.choice(publisher_rules[condition][attr][k])
-
-                        if attr == "staff_information" or attr == "identities":
-                            successful_random_publisher = random.choice(publisher_rules[condition][attr])
+                        successful_random_publisher = random.choice(publisher_rules[condition][attr][k])
```

Comment:
@fiji-flo you're sure this does not break anything even when you run the e2e several times?

Reply:
@fiji-flo

Reply:
@fiji-flo can you comment on this or close?